apps/src/lab2/views/Lab2Wrapper.tsx

```diff
@@ -78,8 +78,15 @@ const Lab2Wrapper: React.FunctionComponent<Lab2WrapperProps> = ({children}) => {
 
   // We duplicate the theme to Lab2Registry, because modals opened via the header (such as the share modal)
   // do not have access to the theme context.
+  // We also update the body class to match the theme, so elements such as the footer update correctly.
```

Comment:
Later I'm hoping we can factor out the footer styling so we don't need to do this - but this seems like a good solution for now!